config-v3-documentation

```diff
@@ -158,14 +170,16 @@ subtasks=@auto
 [limits]
 # Resource limits for various programs: all of them are called KEY_LIMIT,
 # where KEY is one of:
+#   tool        Pisek tools
 #   in_gen      Input generator
 #   checker     Checker
 #   judge       Judge
 #   solve       Primary solution
 #   sec_solve   Secondary solution usually subject to much less strict limits.
 # and LIMIT is one of:
 #   time_limit  Execution time limit [seconds]
-#   clock_limit Wall clock time limit [seconds]
+#   clock_mul   Wall clock multiplier [1]
+#   clock_min   Wall clock minimum [second]   
```

Comment:
It isn't clear what the multiplier multiplies.

```suggestion
#   clock_mul   Wall clock time limit, as a multiple of the time limit
#   clock_min   Wall clock time limit minimum, regardless of the time limit [second]   
```

Reply:
Is not the description on lines 191-192 not enogh?

Also it's not regardles because zero behaves wierd.

Reply:
Sorry, I missed that.